src/components/organisms/LendingCard/LendingCard.tsx

```diff
@@ -154,10 +155,15 @@ export const LendingCard = ({
 
     const { data: availableToBorrow } = useBorrowableAmount(address, currency);
 
+    const orderSideOptions = Object.values(OrderSideMap).map(option => ({
```

Comment:
this is a fixed object (independent of the component). Best to take it out to not create it on every rerender.

Reply:
Ok I will place it before the component is defined